### DISCUSSION TITLE
3.10.2 empty folder?

### DISCUSSION THREAD

**Discussion by Blady-Com** [jralls/gtk-osx-build#76] [Q&A] [answered]
At the beginning of `gtk-osx-setup.sh`, a folder named `3.10.2` is created in the same folder where `gtk-osx-setup.sh` is.
But I can't find out from whom it is created.
The folder remains empty.
What is this folder?

**Comment by jralls** [comment, marked as answer]
When I tested just now it created the directory in $HOME which was the $PWD. `gtk-osx-setup.sh` was in the gtk-osx checkout directory. 

That's the currently configured python version and the first thing that gtk-osx-setup.sh does is clone and install pyenv. I suspect that pyenv's install script creates the directory in $PWD before it redirects to $PYENV_ROOT.
